python-test/features/steps/login_ui.py

```diff
@@ -0,0 +1,74 @@
+from selenium import webdriver
+from behave import given, when, then
+from test_config import TestConfig
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from hamcrest import *
+
+configs = TestConfig.configs()
+base_orb_url = "https://" + configs.get('orb_address')
+user_email = configs.get('email')
+user_password = configs.get('password')
+
+
+@given("that the user is on the orb page")
+def orb_page(context):
+    """Open the required page on Chrome browser
+    Args:
+        context (behave.runner.Context): object that hold contextual information during the running of tests
+    """
+    go_to_page(base_orb_url, context)
+
+
+@when("the Orb user logs in Orb UI")
+def use_credentials(context):
+    """
+    Set the credentials for login in Orb
+    :param context: Behave class that contains contextual information during the running of tests.
+    """
+    enter_information("input-email", user_email, context)
+    enter_information("input-password", user_password, context)
+    context.driver.find_element_by_css_selector(str(".appearance-filled")).click()
+
+
+@then("the user should have access to orb home page")
+def check_home_page(context):
+    WebDriverWait(context.driver, 10).until(EC.url_to_be(f"{base_orb_url}/pages/home"), message="user not enabled to "
+                                                                                                "access orb home page")
+
+
+def go_to_page(page, context):
+    """Open the page in Chrome browser
+    Args:
+        page (string): site's URL
+        context (behave.runner.Context): object that hold contextual information during the running of tests
+    """
+    options = webdriver.ChromeOptions()
+    options.add_argument('--ignore-ssl-errors=yes')
+    options.add_argument('--ignore-certificate-errors')
+    options.add_argument("--start-maximized")
+    context.driver = webdriver.Chrome(options=options)
+    context.driver.get(str(page))
+    assert_that(context.driver.current_url, equal_to(f"{base_orb_url}/auth/login"), "user not enabled to access orb "
+                                                                                    "login page")
+
+
+def enter_information(html_information_name, information, context, source_by=0):
```

Comment:
Looks like `enter_information` and `go_to_page` can be used in many use-cases, not only in login.
Let's create a `ui_utils.py` module and add all functions that are useful to multiple UI-related scenarios.

Also, let's stick with UI nomenclature. So `input_text` is more suitable than `enter_information`, what do you think?

Reply:
I liked. Good idea. Thanks